packages/tailwindcss-react-aria-components/src/index.js

```diff
@@ -0,0 +1,50 @@
+import plugin from 'tailwindcss/plugin';
+
+const attributes = {
+  boolean: [
```

Comment:
I noticed some of these are default variants in Tailwind, so we may want to ensure we aren't removing any expected behavior for these: [`disabled`](https://tailwindcss.com/docs/hover-focus-and-other-states#disabled), [`empty`](https://tailwindcss.com/docs/hover-focus-and-other-states#empty), [`indeterminate`](https://tailwindcss.com/docs/hover-focus-and-other-states#indeterminate), [`required`](https://tailwindcss.com/docs/hover-focus-and-other-states#required), [`open`](https://tailwindcss.com/docs/hover-focus-and-other-states#open-closed-state), [`invalid`](https://tailwindcss.com/docs/hover-focus-and-other-states#invalid).

Reply:
@devongovett beat me to it: https://github.com/adobe/react-spectrum/pull/4933#discussion_r1298678822